src/widgets/JSONForm/JSONForm.js

```diff
@@ -125,15 +126,19 @@ class FocusController {
   };
 }
 
-export const JSONForm = React.forwardRef(
+// The underlying Form component takes a prop formData which just seeds the component
+// but does not update the form after that point.
+// So, just never re-render this component.
+const propsAreEqual = () => true;
```

Comment:
controversial? Just never update this component- if it doesn't update when passing a new form data none of the other props are really gonna change, so..

Reply:
gosh. well, sure - if it doesn't update